scripts/cut_release_or_hotfix_branch.py

```diff
@@ -12,16 +12,27 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Helper script used for creating a new release branch on GitHub.
+"""Helper script used for creating a new release or hotfix branch on GitHub.
 
 ONLY RELEASE COORDINATORS SHOULD USE THIS SCRIPT.
 
 Usage: Run this script from your oppia root folder:
 
-    python -m scripts.cut_release_branch --version="x.y.z"
+For release branch:
+
+    python -m scripts.cut_release_or_hotfix_branch --version="x.y.z"
 
 where x.y.z is the new version of Oppia, e.g. 2.5.3.
+
+For hotfix branch:
+
+    python -m scripts.cut_release_or_hotfix_branch --version="x.y.z"
+    --hotfix_version=d
+
+where x.y.z is the new version of Oppia, e.g. 2.5.3,
+d is number of the hotfix being created.
```

Comment:
Maybe add an example of the generated branch name here and above. That might be useful for a reader.

Reply:
Done.